playground/mocks/data/idp/idx/enroll-profile-submit.json

```diff
@@ -1,84 +1,84 @@
-            "form":{
-              "value":[
+            "name": "userProfile",
+            "form": {
+              "value": [
                 {
-                  "name":"favoriteSport",
-                  "label":"Favorite sport",
-                  "required":true,
-                  "minLength":1,
-                  "maxLength":50
+                  "name": "favoriteSport",
+                  "label": "Favorite sport",
+                  "required": true,
+                  "minLength": 1,
+                  "maxLength": 50
                 },
                 {
-                  "name":"newAttribute",
-                  "label":"Custom attribute",
-                  "required":true,
-                  "minLength":0
+                  "name": "newAttribute",
+                  "label": "Custom attribute",
+                  "required": true,
+                  "minLength": 0
                 }
               ]
             }
           },
           {
-            "name":"stateHandle",
-            "required":true,
-            "value":"02m5bP1_PZfgeox0qFcNeh5Zr71MB3gs3tw7kCsL2G",
-            "visible":false,
-            "mutable":false
+            "name": "stateHandle",
+            "required": true,
+            "value": "02m5bP1_PZfgeox0qFcNeh5Zr71MB3gs3tw7kCsL2G",
+            "visible": false,
+            "mutable": false
           }
         ],
-        "accepts":"application/json; okta-version=1.0.0"
+        "accepts": "application/json; okta-version=1.0.0"
       }
     ]
   },
-  "user":{
-    "type":"object",
-    "value":{
-      "id":"00u13f9y2r7d4zB390g4",
+  "user": {
+    "type": "object",
+    "value": {
+      "id": "00u13f9y2r7d4zB390g4",
       "identifier": "testUser@okta.com"
     }
   },
-  "cancel":{
-    "rel":[
+  "cancel": {
+    "rel": [
       "create-form"
     ],
-    "name":"cancel",
-    "href":"http://localhost:3000/idp/idx/cancel",
-    "method":"POST",
-    "produces":"application/ion+json; okta-version=1.0.0",
-    "value":[
+    "name": "cancel",
+    "href": "http://localhost:3000/idp/idx/cancel",
+    "method": "POST",
+    "produces": "application/ion+json; okta-version=1.0.0",
+    "value": [
       {
-        "name":"stateHandle",
-        "required":true,
-        "value":"02m5bP1_PZfgeox0qFcNeh5Zr71MB3gs3tw7kCsL2G",
-        "visible":false,
-        "mutable":false
+        "name": "stateHandle",
+        "required": true,
+        "value": "02m5bP1_PZfgeox0qFcNeh5Zr71MB3gs3tw7kCsL2G",
+        "visible": false,
+        "mutable": false
       }
     ],
-    "accepts":"application/json; okta-version=1.0.0"
+    "accepts": "application/json; okta-version=1.0.0"
   },
-  "app":{
-    "type":"object",
-    "value":{
-      "name":"okta_enduser",
-      "label":"Okta Dashboard",
-      "id":"0oay6fLeui13rEyTj0g3"
+  "app": {
+    "type": "object",
+    "value": {
+      "name": "okta_enduser",
+      "label": "Okta Dashboard",
+      "id": "0oay6fLeui13rEyTj0g3"
     }
   }
 }
```

Comment:
@fangshi-okta this is the file I was talking about. I just formatted the file without changing the content, thanks.

Reply:
Thank you!